Xamarin.Forms.Core/Slider.cs

```diff
@@ -28,16 +28,18 @@ public class Slider : View, IElementConfiguration<Slider>
 			return value;
 		});
 
-		public static readonly BindableProperty ValueProperty = BindableProperty.Create("Value", typeof(double), typeof(Slider), 0d, BindingMode.TwoWay, coerceValue: (bindable, value) =>
+		public static readonly BindableProperty ValueProperty = BindableProperty.Create("Value", typeof(double), typeof(Slider), 0d, BindingMode.TwoWay, (bindable, value) =>
+		{
+			var slider = (Slider)bindable;
+			return (double)value >= slider.Minimum && (double)value <= slider.Maximum;
+		}, coerceValue: (bindable, value) =>
 		{
 			var slider = (Slider)bindable;
 			return ((double)value).Clamp(slider.Minimum, slider.Maximum);
 		}, propertyChanged: (bindable, oldValue, newValue) =>
```

Comment:
If you remove validate, then you'll face bug 41960. Coerce will set Value to 100 (user intends 150). If you keep validate, then the user knows to update Minimum and Maximum first before setting Value to 150.